src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -105,13 +107,31 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     showAppliedFilters: false,
   };
 
+  @observable cellSizes: number[];
+  get persistentCellSizes(): number[] {
+    const { clusterId, tableId, renderTableHeader: { length } } = this.props;
+    const savedSizes = userStore.getTableSizing(clusterId, tableId);
+
+    return savedSizes.length !== length ? new Array<number>(length) : savedSizes;
+  }
+
+  set persistentCellSizes(sizes: number[]) {
+    const { clusterId, tableId } = this.props;
+  
+    userStore.setTableSizing(clusterId, tableId, sizes);
```

Comment:
Maybe better to work with `clusterStore` instead to persist preferences? Therefore would no need in introducing `props.clusterId` to `ItemsListLayout` since you can get `getHostedClusterId()` from `cluster-store.ts` to manage proper place to keep in cluster-state.

Reply:
My logic was that column sizes are more of a fit for user preferences.

Reply:
Would it make sense for an initial version to be "global" or use the same column widths over all clusters? @ixrock what do you think?

Reply:
I thought having resized columns on one table then to see same result being applied to unrelated tables would be annoying for users. Besides implementing independent sizing for each particular table in each cluster didn't seem too difficult.

Reply:
I meant the same table type has the same settings, not every table has the same settings. So I think we are on the same page.